### PR TITLE
Delete Account before terms acceptance

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This PR adds a button to delete one's account in the "Verify email" page and in the "Terms"page when creating a new account. Should be used in conjunction with https://github.com/ibi-group/otp-middleware/pull/213.

## PR Checklist
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

Screenshots of the changes:

Verify email page:
image

Terms page:
image

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks great! One quick suggestion

File: lib/components/user/verify-email-pane.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          onClick={handleResend}
+          style={{ padding: '0px' }}
+        >
+          <FormattedMessage id="components.VerifyEmailPane.resendVerification" />
+        </Button>
+      </div>
+
+      <div style={{ marginTop: '3em' }}>
+        <DeleteUser size="large" />
+      </div>
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply lib/components/user/verify-email-pane.tsx:71]: Is this to make these buttons the same width?

[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply lib/components/user/verify-email-pane.tsx:71]: @amy-corson-ibigroup I am not sure I like this layout. It is not consistent with the Terms screen, which follows this screen, and where the "Delete my account" button is on the left side (and the "Next" button to the right).

[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply lib/components/user/verify-email-pane.tsx:71]: I see your point, in that case I'm happy to merge now and we can revisit the layout another time!

**Pull request merged**